### PR TITLE
BZ1907885: clarify what configuration is copied by --copy-network

### PR DESCRIPTION
**Pull request by miabbott**

### CODE REVIEW

**Comment by miabbott** [comment]
cc: @dustymabe @bobfuru @bgilbert

**Comment by bobfuru** [comment]
LGTM, let me know when you're ready for a merge. Thanks!

File: modules/installation-user-infra-machines-advanced.adoc [hunk 1]
```diff
@@ -50,6 +50,11 @@ system using available RHEL tools, such as `nmcli` or `nmtui`.
 $ coreos-installer install --copy-network \
      --ignition-url=http://host/worker.ign /dev/sda
 ----
++
+[IMPORTANT]
+====
+The `--copy-network` option will only copy configuration files found under `/etc/NetworkManager/system-connections`.
```
Comment (bgilbert) [inline modules/installation-user-infra-machines-advanced.adoc:56]: What about something like:

```suggestion
The `--copy-network` option only copies networking configuration found under `/etc/NetworkManager/system-connections`. In particular, it does not copy the system hostname.
```

That could help users who don't know where the hostname lives.

**Review by bgilbert — COMMENTED** [review]
The original bug also asks for docs on configuring the hostname via the Ignition pointer config, but I don't know if that's meant to be in-scope here.

File: modules/installation-user-infra-machines-static-network.adoc [hunk 2]
```diff
@@ -197,6 +197,11 @@ a|Delete a default kernel argument from the installed system.
 
 a|`-n`, `--copy-network`
 a|Copy the network configuration from the install environment.
++
+[IMPORTANT]
+====
+The `--copy-network` option will only copy configuration files found under `/etc/NetworkManager/system-connections`.
```
Comment (bgilbert) [inline modules/installation-user-infra-machines-static-network.adoc:203]: Also needs update.

**Review by bgilbert — APPROVED** [review]
LGTM

**Comment by miabbott** [comment]
Updated per Benjamin's feedback; should be ready to merge.

This can be cherry-picked back to `enterprise-4.6`

**Pull request merged**